src/kernel/caches.h

```diff
@@ -16,6 +16,10 @@ static constexpr size_t MAX_BLOCK_DB_CACHE{2_MiB};
 //! Max memory allocated to coin DB specific cache (bytes)
 static constexpr size_t MAX_COINS_DB_CACHE{8_MiB};
 
+static constexpr size_t MIN_DB_CACHE_BATCH{16_MiB};
```

Comment:
In commit "coins: introduce dynamic batch size calculator based on dbcache value" (b249239c1747a5425e5161974d2b9c9c5be17dab)

Name suggests this is a minimum size, but value is actually ignored if `-dbbatchsize` is specified, so is not an enforced minimum. Would suggest calling it a "minimum default" value instead of a "minimum" value to be more accurate. Or it might be better to just drop `MIN_DB_CACHE_BATCH` and `MAX_DB_CACHE_BATCH` here and just inline these in the `GetDbBatchSize` function since they are not used anywhere outside it.